### ISSUE TITLE
Severe bug in iOS app version 4.7: display is unstable and unusable 

### ISSUE THREAD

**Issue by kevinp2** [osmandapp/OsmAnd-iOS#3618]
### Description

The map does not remain steady in orientation and zoom when the **Map orientation** is **Movement direction**.

This bug is new with the new app version.  The older version did not have it with the same iPhone

### Steps to reproduce

Drive anywhere in a car with or without a route.   Change the **Map orientation** to **Movement direction** (which is the most useful map mode).

### Actual result

The display rotates and zooms in and out randomly.  It makes the app unusable.

https://github.com/osmandapp/OsmAnd/assets/609677/82a20d76-7284-4f33-8fdf-db76dc9422a5

### Expected result

A steady map display like in the previous version.

### Your Environment (required)

```
OsmAnd Version: 4.7
Android/iOS version: 17.4.1
Device model: iPhone 15 Pro Max

```

**Comment by jbcormoreche** [comment]
Do I need to open a different issue for the android version as it was marked as an ios bug?

**Comment by sanketgarade** [comment]
Same with CarPlay #3619

**Comment by kevinp2** [comment]
Confirm that this is fixed in OSMAnd 4.7.4
